utils/debug.c

```diff
@@ -401,10 +401,8 @@ void print_diff_percent(uint64_t base_nsec, uint64_t pair_nsec)
 	percent = 100.0 * (int64_t)(pair_nsec - base_nsec) / base_nsec;
 
 	/* for some error cases */
-	if (percent > 999.99)
+	if (percent > 999.99 || percent < -999.99)
 		percent = 999.99;
-	else if (percent < -999.99)
-		percent = -999.99;
```

Comment:
Is this really correct? If `percent` is less than `-999.99`, it used to be `-999.99` but it makes it to `999.99`, which is different.

Reply:
Oh my, Thank you for the code review @honggyukim :)